htlcswitch/link.go

```diff
@@ -2349,6 +2335,15 @@ func (l *channelLink) htlcSatifiesPolicyOutgoing(policy ForwardingPolicy,
 		return &lnwire.FailExpiryTooFar{}
 	}
 
+	// Check to see if there is enough balance in this channel.
+	if amt > l.Bandwidth() {
```

Comment:
previous comparison used `>=`

Reply:
If was `bandwidth < amt` and now it is `amt > bandwidth`. Identical?

Reply:
got it i see now, prior one used non-strict inequality for success case, and here we use the strict inequality for the failure case